### DISCUSSION TITLE
unleash-proxy-client in React lib is intercepting uncaught exceptions

### DISCUSSION THREAD

**Discussion by westandy** [Unleash/unleash#3490] [Q&A]
![Screenshot](https://user-images.githubusercontent.com/104597263/230949476-40a906ab-339c-47e2-a19c-f47ba5483fa3.png)

The uncaught exception is within libraries built for my company's application, and has nothing to do with Unleash.   But this display overlays the screen, and looking at the stack trace on the left, many people immediate see this as an Unleash problem.   Is there a way to disable this from happening?  

The libraries I am using are:
```
    "@unleash/proxy-client-react": "3.5.2",
    "unleash-proxy-client": "2.4.3",
```

**Comment by pransh15** [comment]
Hey @westandy 👋
Thanks for reporting this on Discussions, looping in our devs to respond to this issue. ➿
